### PR TITLE
New package: quotient-im.Quaternion version 0.0.95.1

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/q/quotient-im/Quaternion/0.0.95.1/quotient-im.Quaternion.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: quotient-im.Quaternion
+PackageVersion: 0.0.95.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Quaternion-0.0.95.1\quaternion.exe
```
Comment (stephengillie) [inline manifests/q/quotient-im/Quaternion/0.0.95.1/quotient-im.Quaternion.installer.yaml:9]:
```suggestion
- RelativeFilePath: Quaternion-0.0.95.1\quaternion.exe
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by Exorcism0666** [comment]
## For moderators

This account is bot account and belongs to CoolPlayLin. Any issue should call him up instead of assigning to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let CoolPlayLin know).

Have a good day!

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification. It has the write permission of this account's fork so that it can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Review by stephengillie — APPROVED** [review]

**Pull request merged**